.github/workflows/build.yml

```diff
@@ -77,13 +77,13 @@ jobs:
           shared-key: "${{ runner.os }}-full-${{ hashFiles('./Cargo.lock') }}"
 
       - name: Cargo Build
-        run: cargo build --workspace --all-targets --all-features ${{ matrix.config.extraArgs }}
+        run: cargo build --workspace --release --all-targets --all-features ${{ matrix.config.extraArgs }}
```

Comment:
when using `debug` builds, `spin up` was taking really long time. so I had a discussion with @radu-matei  and he suggested we can go for the release build here and switch to debug builds on case by case basis when required.

about the path, I should have changed that as well to reflect that it is a release build now. the reason it does not give error is because in I am downloading the `spin` artifact in the `debug` directory itself. I will change this.

Reply:
changed it to use `target/release` directory